R/loopy.R

```diff
@@ -55,12 +55,18 @@ loopy <- function(resamples, grid, static) {
     # --------------------------------------------------------------------------
     # Iterate over model parameters
 
+    # Make a copy of the current workflow so that we can finalize it multiple
+    # times
+    pre_wflow <- current_wflow
```

Comment:
I figured out that the `finalize_*()` functions will not update parameters whose values currently are `tune()`. We save this to maintain those values.